framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -256,13 +256,13 @@ sWscAttrVendorExtension:
     _value: WSC_VENDOR_ID_WFA_3
   subelement_id:
     _type: uint8_t
-    _value: 0x6
+    _value: 0x0
   subelement_length:
     _type: uint8_t
     _value: 0x1
   subelement_value:
     _type: uint8_t
-    _value: TEARDOWN
+    _value: 0x20
```

Comment:
Why did you replace the enum with value?
And why this value is defined as the default value?

Reply:
> Why did you replace the enum with value?

 Good point... will fix.

> And why this value is defined as the default value?

It is the only allowed value for the WFA-WSC Version IE. But maybe you mean it should be `_const_value`? Then you're probably right, but this entire yaml file should use `_const_value`.

Reply:
_value_const: TEARDOWN
https://github.com/prplfoundation/prplMesh/tree/master/framework/tlvf#classes